extensions/amp-consent/0.1/amp-consent.js

```diff
@@ -132,6 +135,8 @@ export class AmpConsent extends AMP.BaseElement {
     // ConsentConfig has verified that there's one and only one consent instance
     this.consentId_ = this.consentConfig_['consentInstanceId'];
 
+    this.matchedGeoGroups_ = this.consentConfig_['matchedGeoGroups'];
```

Comment:
it's a bit weird to get that from the merged `config` JSON.
`ConsentConfig` should have a public method to return that.

Reply:
Should we also do the same for `consentInstanceId `? Or does it make sense to pull that directly from the config? I figure the latter.